packages/headless/src/test/mock-commerce-facet-request.ts

```diff
@@ -0,0 +1,18 @@
+import {CommerceFacetRequest} from '../features/commerce/facets/facet-set/interfaces/request';
+
+export function buildMockCommerceFacetRequest(
+  config: Partial<CommerceFacetRequest> = {}
+): CommerceFacetRequest {
+  return {
+    facetId: '',
+    field: '',
+    type: 'regular',
+    numberOfValues: 8,
+    values: [],
+    isFieldExpanded: false,
+    freezeCurrentValues: false,
+    preventAutoSelect: false,
```

Comment:
This relates to a previous comment I made, but do we need to pass freezeCurrentValues and preventAutoSelect? I thought the product listing API didn't support those properties.